src/repos.json

```diff
@@ -0,0 +1,46 @@
+{
+  "rhel-86": [
```

Comment:
Let's use the constant [RHEL_8] here, we'll only ever support rhel-8, so might as well.